Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN \
   curl -L -o sbt-$SBT_VERSION.deb http://dl.bintray.com/sbt/debian/sbt-$SBT_VERSION.deb && \
   dpkg -i sbt-$SBT_VERSION.deb && \
   rm sbt-$SBT_VERSION.deb && \
+  apr-get install openjdk-8-jdk && \
```

Comment:
Hi, thanks for the PR. However, the base image is already `openjdk:8` here. Is this still needed?